README.md

```diff
@@ -20,6 +20,10 @@ Each tasks wraps the existing [Power Platform CLI](https://aka.ms/PowerPlatformC
 
 Please use the issues tracker in the home repo: <https://github.com/microsoft/powerplatform-build-tools/issues>
 
+## Community Office Hours
+
```

Comment:
Add a quick description - for example: We hold monthly office hours (every 3rd thursday of the monty) where you can ask specific question to product team.